users/vieting/experiments/switchboard/transducer/feat/data.py

```diff
@@ -207,6 +207,13 @@ def get_returnn_ogg_datasets(**kwargs) -> Dict[str, Dict]:
         "dev": returnn_datasets["dev"].get_data_dict()["datasets"]["ogg"],
         "eval_datasets": {"devtrain": returnn_datasets["eval_datasets"]["devtrain"].get_data_dict()["datasets"]["ogg"]},
     }
+    if use_multi_proc_dataset:
+        returnn_datasets["train"] = {
+            "class": "MultiProcDataset",
+            "dataset": returnn_datasets["train"].get_data_dict()["datasets"]["ogg"],
```

Comment:
Did you test this? `get_data_dict()` was already called above, so it should already be a dict.